docs/workflows/api-token.md

```diff
@@ -288,3 +288,86 @@ title="/tokens"
   "total_size": 1
 }
 ```
+
+## Custom Claims
+
+The `custom_claims` parameter allows for additional information to be stored
+within a token. Any data may be associated with a token by using the
+custom_claims URL parameter. The provided data must be a valid JSON object. All
+fields are accessible, when parsed from a token within the `bi_custom` field of
+the JWT payload. `
+
+<MultiLanguageCodeBlock
+curl='curl "https://auth-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/applications/$(APPLICATION_ID)/token?principal_type=application&principal_id=$(APPLICATION_ID)&
+-u "$(MANAGEMENT_API_CLIENT_ID):$(MANAGEMENT_API_CLIENT_SECRET)" --basic \
+-H "Content-Type: application/x-www-form-urlencoded" \
+-d "grant_type=client_credentials&scope=$(SCOPES)&custom_claims=$(CLAIMS_JSON_OBJECT)'
```

Comment:
I think we should add a note below this curl request saying that:

> In this case, we set the `CLAIMS_JSON_OBJECT` to `{"a": "b", "c": "d"}`

Reply:
Fixed. I added the text below the example response.